telegram-service/src/app.update.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Command,
+  Ctx,
+  Hears,
+  Start,
+  Update,
+  Sender,
+  Action,
+} from 'nestjs-telegraf';
+import { Context } from './interfaces/context.interface';
+import { SceneNames, Actions, Commands } from './app.constants';
+
+@Update()
+export class AppUpdate {
+  supportedCommands = [
+    { command: '/help', description: 'Help' },
+    { command: '/start', description: 'Start' },
+  ];
+
```

Comment:
move it to constants file and import it here

Reply:
> move it to constants file and import it here

++